learn/cookbooks/qovery.md

```diff
@@ -1,38 +1,64 @@
 # Deploy a Meilisearch instance on Qovery
 
-_The following is a guest post by Romaric Philogène, CEO and Co-Founder of Qovery._
+[Qovery](https://www.qovery.com) is a fully-managed cloud platform that runs on your AWS, Digital Ocean and Scaleway account where you can host static sites, backend APIs, databases, cron jobs, and all your other apps in one place.
 
-## Introduction
+Qovery provides **free hosting** for individual developers and includes the following features:
 
-In this tutorial I explain how to deploy a pre-configured Meilisearch instance in one click.
+* Continuous, automatic builds & deploys from GitHub and GitLab
+* Automatic SSL certificates through [Let's Encrypt](https://letsencrypt.org)
+* Free SSD storage
+* Unlimited collaborators
+* Unlimited [custom domains](https://hub.qovery.com/docs/using-qovery/configuration/application/#domains)
 
-_Qovery provides free Cloud hosting with databases, SSL, a global CDN, and automatic deploys with Git._
+## Setup
 
-Deploying Meilisearch with Qovery provides:
+### 1. Create a Qovery account
 
-- A pre-configured Meilisearch instance.
-- A free SSD storage.
-- A free SSL.
-- Optional: custom domain.
+Visit the [Qovery dashboard](https://start.qovery.com) to create an account if you don't already have one.
 
-## Step-by-step Meilisearch deployment
+### 2. Create a project
 
-### 1. Create a Qovery Account
+* Click on **Create project** and give a name to your project
+* Click on **Next**
 
-Visit [the Qovery dashboard](https://start.qovery.com) to create an account if you don't already have one.
+!["Create Project" button](https://hub.qovery.com/img/heroku/heroku-2.png)
 
-### 2. Create a project
+### 3. Create a new environment
+
+* Click on **Create environment** and give it a name (e.g. staging, production).
+
+!["Create environment" button](https://hub.qovery.com/img/heroku/heroku-3.png)
```

Comment:
As the current guardian of alt text would you make any changes to this @maryamsulemani97 ?

Reply:
No, I followed this format for other similar images. Ik it's not a good practice for the alt text to repeat what your content says, but until we think of a better alternative(maybe reword the actual text), I'd like to stay consistent